packages/search/app/styles/navi-search/components/navi-search-bar.scss

```diff
@@ -97,7 +97,7 @@
     }
 
     &--row:nth-child(even) {
-      background: rgba($color-brand-500, 0.05);
+      background: rgba(map-get($denali-brand-colors, '500') 0.05);
```

Comment:
I think a comma is missing here background: rgba(map-get($denali-brand-colors, '500'), 0.05);

Reply:
```suggestion
      background: rgba(map-get($denali-brand-colors, '500'), 0.05);
```